packages/dependency-royalty-management/contracts/RoyaltyDistributer.sol

```diff
@@ -14,16 +14,13 @@ contract RoyaltyDistributer is IERC2981Upgradeable {
 
     /// @notice Returns how much royalty is owed and to whom based on ERC2981
     /// @dev tokenId is one of the EIP2981 args for this function can't be removed
-    /// @param _tokenId of catalyst for which the royalty is distributed
     /// @param _salePrice the price of catalyst on which the royalty is calculated
```

Comment:
We probably don't want to say `price of catalyst` right, since this contract is a dependency it can be used anywhere.